desktop/wallet/package.json

```diff
@@ -47,8 +47,9 @@
   },
   "update": {
     "repositoryRootUrl": "https://api.github.com/repos/burst-apps-team/phoenix",
-    "checkIntervalMins": 1,
-    "tagPrefix": ""
+    "certFingerprint": "31:11:50:0C:4A:66:01:2C:DA:E3:33:EC:3F:CA:1C:9D:DE:45:C9:54:44:0E:7E:E4:13:71:6B:FF:36:63:C0:74",
```

Comment:
this one is the github ssl cert fingerprint...we should be quite secure garantueeing original source. Without the fingerprint check, I was able to create a self-signed ssl certificate which passed validation.

Reply:
good find!